packages/cw721/src/state.rs

```diff
@@ -161,12 +177,62 @@ where
 pub struct CollectionInfo {
     pub name: String,
     pub symbol: String,
+    pub updated_at: Timestamp,
+}
+
+/// Explicit type equivalent to `Vec<Attribute>`, for better distinction.
+pub type CollectionExtensionAttributes = Vec<Attribute>;
+
+#[cw_serde]
+pub struct Attribute {
+    pub key: String,
+    pub value: Binary,
+}
```

Comment:
This is the new optional `CollectionExtensionAttributes`. Any arbitrary collection data can be added by created into this list.